### DISCUSSION TITLE
Diferença -o ou -n ao criar o projeto?

### DISCUSSION THREAD

**Discussion by victor-sc-92** [balta-io/forum#263] [Tira Dúvidas] [answered]
Aula: [.Net/Tipos de projeto](https://balta.io/player/assistir/5ef614da-366d-6156-d049-694800000000/0482ee81-8f83-4653-a421-000028010404)

 Na aula se usa o comando:
`$ dotnet new console -o NovoTeste`

Em alguns tutoriais eu vi usarem o parâmetro -n no lugar do -o:
`$ dotnet new console -n NovoTeste`

Não senti diferença entre eles, existe algum que seja mais indicado ou mais usado?
Qual a diferença prática entre eles?
Quando devo usar um ou outro?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @CodeAkio, como vai?
Até onde sei, nenhuma... `-o` vem de Output, que vai gerar uma pasta nova com a aplicação dentro. Como o App sempre tem como nome e namespace raiz o nome da pasta, então ele vai receber o nome que informar.

O `-n` é de name, que faz exatamente o mesmo processo acima.
